bundle/manifests/websphere-liberty.clusterserviceversion.yaml

```diff
@@ -44,17 +44,17 @@ metadata:
       ]
     capabilities: Auto Pilot
     categories: Application Runtime
-    operators.operatorframework.io/builder: operator-sdk-v1.6.1+git
+    operators.operatorframework.io/builder: operator-sdk-v1.17.0+git
```

Comment:
@leochr  As far as I know, this copy is just a convenience copy of this file. Any build of our product will regenerate this file, at this exact path,  with whatever version of the operator-sdk is available at that time.  

I don't have the defect handy, but also I had moved to the latest stable version to address a bug I was hitting in OLM when working on the twas operator,  so we might need to discuss what is needed and investigate the backwards compatibility of running with a different operator-sdk than we originally generated with.

Reply:
@smcclem It is a convenience copy, but it's used locally when the bundle is built, run and tested. Since different operator-sdk versions can generate the artifacts differently, we should use the same operator-sdk, both locally and in build, to keep it consistent. We'll also avoid merge conflicts due to our PRs containing different values for metadata (like this one) because of different operator-sdk versions.

Please provide the details around the reason(s) for having to move to a newer operator-sdk version. The OLO ad WLO are at operator-sdk 1.6.4. Moving up the operator-sdk requires certain manual migration steps we would need to follow. Thanks